debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SnapshotIT.java

```diff
@@ -84,6 +84,11 @@ public void takeSnapshotInSnapshotMode() throws Exception {
         takeSnapshot(SnapshotLockingMode.SNAPSHOT);
     }
 
+    @Test
+    public void takeSnapshotInConsistentMode() throws Exception {
```

Comment:
Seems one for `EXCLUSIVE` is missing, too?

Reply:
It's in line 78.

Reply:
Ah yes. Of course.